packages/backend/src/apps/postman/actions/send-transactional-email/index.ts

```diff
@@ -152,15 +154,38 @@ const action: IRawAction = {
     const blacklistedRecipients = dataOut.recipient.filter(
       (_, i) => dataOut.status[i] === 'BLACKLISTED',
     )
+
+    /**
+     * Send blacklist notification email if any
+     */
+    if (blacklistedRecipients.length > 0 && !$.execution.testRun) {
+      try {
+        await sendBlacklistEmail({
+          flowId: $.flow.id,
+          flowName: $.flow.name,
+          userEmail: $.user.email,
+          executionId: $.execution.id,
+          blacklistedRecipients,
+        })
+      } catch (e) {
+        logger.error(e)
```

Comment:
nit: you're already storing `e` as `error` prop in the 2nd logger call

Reply:
somehow the second logger call doesnt log the full error. will leave this here for the time being and remove when it's stable (famous last words)